spec/paperclip/thumbnail_spec.rb

```diff
@@ -480,6 +480,39 @@ def to_s
         assert_equal "50x50", `#{cmd}`.chomp
       end
     end
+    
+    context "with a specified frame_index" do
+      before do
+        @thumb = Paperclip::Thumbnail.new(@file,
+                                          geometry: "50x50",
+                                          frame_index: 5,
+                                          format: :jpg,
+                                          )
+      end
+
+      it "creates the thumbnail from the frame index when sent #make" do
+        @thumb.make
+        assert_equal @thumb.frame_index, 5
+      end
+    end
+    
+    context "with a specified frame_index out of bounds" do
+      before do
+        @thumb = Paperclip::Thumbnail.new(@file,
+                                          geometry: "50x50",
+                                          frame_index: 20,
+                                          format: :jpg,
+                                          )
```

Comment:
@tute can you help me out with what Hound wants here?  I have tried a few different options for aligning the ( ), but it keeps reporting errors